kubedifflib/_diff.py

```diff
@@ -79,7 +97,7 @@ def diff_dicts(path, want, have):
 
 
 def normalize(value):
-  if isinstance(value, int):
+  if isinstance(value, (int, float)):
```

Comment:
I think we can do `isinstance(value, numbers.Number)` (as per https://docs.python.org/2/library/numbers.html)